File: app/src/main/java/com/androidevlinux/percy/UTXO/data/models/coinmarketcap/Metadata.java
package com.androidevlinux.percy.UTXO.data.models.coinmarketcap;


import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Metadata {

    @SerializedName("timestamp")
    @Expose
    private Integer timestamp;
    @SerializedName("error")
    @Expose
    private Object error;

    public Integer getTimestamp() {
        return timestamp;
    }

    public void setTimestamp(Integer timestamp) {
        this.timestamp = timestamp;
    }

    public Object getError() {
        return error;
    }

    public void setError(Object error) {
        this.error = error;
    }

}

File: app/src/main/java/com/androidevlinux/percy/UTXO/data/models/coinmarketcap/Metadata.kt
package com.androidevlinux.percy.UTXO.data.models.coinmarketcap


import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Metadata {

    @SerializedName("timestamp")
    @Expose
    var timestamp: Int? = null
    @SerializedName("error")
    @Expose
    var error: Any? = null

}
